build/templates/setup.py.mako

```diff
@@ -46,7 +46,7 @@ setup(
     extras_require={
         'grpc': [
             'grpcio>=1.59.0,<2.0',
-            'protobuf>=4.21.6,<5.0'
+            'protobuf>=4.21.6'
```

Comment:
I'm not sure if this will work. We may need to bump it to 5.27.2, based on the failure to import nifake with protobuf 5.26.1.